scriptworker/constants.py

```diff
@@ -36,16 +37,20 @@
     "max_connections": 30,
     "credential_update_interval": 300,
     "reclaim_interval": 300,
+    "poll_git_interval": 300,
```

Comment:
Sure.  Comments are the main reason why I'm considering yaml instead of json for the config files, but `constants.py` can take comments in the meantime.